api/envoy/extensions/common/async_files/v3/async_file_manager.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package envoy.extensions.common.async_files.v3;
+
+import "xds/annotations/v3/status.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.common.async_files.v3";
+option java_outer_classname = "AsyncFileManagerProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/common/async_files/v3;async_filesv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+option (xds.annotations.v3.file_status).work_in_progress = true;
+
+// [#protodoc-title: AsyncFileManager configuration]
+
+// [#extension: envoy.common.support.async_files]
```

Comment:
Sorry for the runaround, but I think this is where everything went sideways for you. Essentially something in the docs is requiring that this have an extension category. The thing is, there shouldn't be any category for this message. This is just a helper message that will get included in other real config (filters, etc.). So I don't think there should be any extension here or category as it's not a real thing. Can you remove this tag and the one below? What error do you get if you do so? If something in the docs is causing you issues, I would just mark this [not-implemented-hide] (or whatever the tag is) until something actually includes this message that we need documented.

Reply:
Done (and yes, once it's de-extensioned the whole cascade of requirements goes away).